MCprep_addon/optimize_scene.py

```diff
@@ -142,12 +159,9 @@ def execute(self, context):
 
 		# Optimizer Settings.
 		Quality = scn_props.quality_vs_speed
-
-		# Render engine settings.
-		# TODO: Add better volumetric optimizations by checking volumetric materials and enabling certain features that benifit the scene (such as homogeneous)
-		if scn_props.volumetric_bool:
-			Samples += 50
-			Volume = 2
+		UseScrambling = scn_props.scrambling_unsafe
```

Comment:
nit about variable naming: `uses_scrambling` here and below (there's some variable names from your prior merge too that I didn't get to updating)